File: Java_Fragoso/src/1-Uniao_Vetores.java
import java.util.Arrays;
import java.util.Scanner;

public class Uniao_Vetores {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        Integer[] A = new Integer[5];
        int C = 0;

        while(C < 5) {
            if(C == 0) {
                System.out.print("Digite um valor: ");
                A[C] = scanner.nextInt();
                C++;
            } else {
                int chave = 0;
                System.out.print("Digite um valor: ");
                int valor = scanner.nextInt();

                for(int i = 0; i < C; i++) {
                    if(A[i] == valor) {
                        chave = 1;
                        break;
                    }
                }


                if(chave == 0) {
                    A[C] = valor;
                    C++;
                } else {
                    System.out.println("Valor já existe. Tente outro valor.");
                }
            }
        }

        System.out.println("Valores armazenados:");
        System.out.println(Arrays.toString(A));
    }
}

File: Python/1-Uniao_Vetores.py
# O algoritmo insere elementos em um vetor A de 5 posições, garantindo que não haja repetição. Ele utiliza uma variável Chave para verificar se o valor já existe no vetor antes de inseri-lo.

def inserir_elementos():
    A = [None] * 5
    C = 0
    
    while C < 5:
        if C == 0:
            A[C] = int(input("Digite um valor: "))
            C += 1
        else:
            Chave = 0
            valor = int(input("Digite um valor: "))
            for i in range(C):
                if A[i] == valor:
                    Chave = 1
                    break
            if Chave == 0:
                A[C] = valor
                C += 1
            else:
                print("Valor já existe. Tente outro valor.")
    return A

A = inserir_elementos()
print(A)

# Exemplo de uso:
# A = inserir_elementos()  # Execute e insira valores manualmente.